Library/Homebrew/rubocops/lines_cop.rb

```diff
@@ -15,6 +15,66 @@ def audit_formula(_node, _class_node, _parent_class_node, _body_node)
           problem ":tex is deprecated" if depends_on?(:tex)
         end
       end
+
+      class ClassInheritance < FormulaCop
+        def audit_formula(_node, class_node, parent_class_node, _body_node)
+          begin_pos = start_column(parent_class_node)
+          end_pos = end_column(class_node)
+          return unless begin_pos-end_pos != 3
+          problem "Use a space in class inheritance: class #{@formula_name} < #{class_name(parent_class_node)}"
+        end
+      end
+
+      class Comments < FormulaCop
+        def audit_formula(_node, _class_node, _parent_class_node, _body_node)
+          # Commented-out cmake support from default template
+          audit_comments do |comment|
+            next unless comment.include?('# system "cmake')
+            problem "Commented cmake call found"
```

Comment:
Is there a reason to still keep this separate from the main `audit_comments` method for comments from the default template? It seems like they could probably be combined. @MikeMcQuaid any opinion here?